Package.swift

```diff
@@ -265,6 +265,12 @@ var targets: [Target] = [
             "TSCclibc": .staticFramework,
             "TSCLibc": .staticFramework,
             "ArgumentParser": .staticFramework,
+            "Mockable": .staticFramework,
+            "MockableTest": .staticFramework,
+        ],
+        targetSettings: [
+            "Mockable": ["ENABLE_TESTING_SEARCH_PATHS": "YES"],
+            "MockableTest": ["ENABLE_TESTING_SEARCH_PATHS": "YES"],
         ],
```

Comment:
I would just add them to the "default" mapping [here](https://github.com/tuist/tuist/blob/a61e6e59a53c62e70fa9233d00cc4806c0f77045/Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift#L437)